pages/expandable-section/permutations.page.tsx

```diff
@@ -13,7 +13,7 @@ import ScreenshotArea from '../utils/screenshot-area';
 /* eslint-disable react/jsx-key */
 const permutations = createPermutations<ExpandableSectionProps>([
   {
-    expanded: [true, false],
+    defaultExpanded: [true, false],
```

Comment:
This change allows to interact with the page for manual testing, even though it will load with the same expanded or collapsed state as before.